lib/simpleRequester.js

```diff
@@ -24,6 +33,13 @@ exports.do = options => {
         }
         reject(HttpErrors(res.statusCode, body.error))
         return
+      } else if ((res.statusCode === 403) &&
+                  (res.request.uri.path.endsWith('/analyses')) &&
+                  (options.ethAddress === trialEthAccount)) {
+        // FIXME: Remove when API gives back a custom message like this:
+        // eslint-disable-next-line prefer-promise-reject-errors
+        reject('Trial user does not allow listing previous analyses. ' +
```

Comment:
Can you please update this to:

*The trial user does not allow listing previous analyses. To enable this feature, please sign up for a free account on: https://mythx.io*